client/reader/user-stream/index.tsx

```diff
@@ -66,13 +66,17 @@ export function UserStream( { userId, requestUser, user, streamKey, isLoading }:
 
 	const renderContent = (): React.ReactNode => {
 		const basePath = currentPath.split( '?' )[ 0 ];
+		const basePathParts = basePath.split( '/' );
+		const isListsPath = basePathParts[ basePathParts.length - 1 ] === 'lists';
```

Comment:
1. Why are we changing this implementation?
2. If its needed then we should make it easier to add new tabs, currently we'll need a new variable.